templates/user-data.tftpl

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+export DEBIAN_FRONTEND=noninteractive
+export RUNNER_CFG_PAT=${personal_access_token}
+
+apt-get -qy update
+apt-get -qy install build-essential gcc jq
+
+svc_user="ghrunner"
+useradd -m $svc_user -s /bin/bash
+cd /home/$svc_user || exit
+
+curl -fsSL https://raw.githubusercontent.com/actions/runner/main/scripts/create-latest-svc.sh -o /tmp/create-latest-svc.sh
```

Comment:
This is nice; might be worth bringing these changes over to `equinix-labs/metal-device-action`...or maybe this is an opportunity to start that library of reusable userdata scripts you were talking about?

Reply:
@cprivitere shared with me the create-latest-svc script, is there any reason not to use it in `equinix-labs/metal-device-action` ?